action.yml

```diff
@@ -12,10 +12,11 @@ runs:
     - uses: actions/stale@v9
       with:
         repo-token: "${{ inputs.TOKEN }}"
-        stale-pr-message: 'This PR is stale because it has been open 30 days with no activity. Remove "Stale" label or comment or this will be closed in 7 days.'
+        # PR stale settings
+        stale-pr-message: 'This PR is stale because it had no activity for the past 30 days. Remove "Stale" label or comment or this PR will be closed in 7 days.'
```

Comment:
Looking at this again, I think the message is a bit ambiguous:
"Remove label or comment or this PR will be closed" can be read as "Remove label or remove comment ...".  The sequence of "or"s is also a bit awkward.

I think the following message is clearer:
"Remove the Stale label or add a comment otherwise this PR will be closed..."

Reply:
Wait.  I'm stupid.  Removing the comment is actually the correct interpretation, right?
So maybe just changing to:
"Remove the stale label or comment otherwise ...".

Reply:
Thanks for the suggestion, I reworded both messages.